### PR TITLE
Set the tags in an pipeline agnostic way.

### PR DESCRIPTION
**Pull request by russoh**
### Purpose of this PR
Correct output of tags for each render pipeline.

---
### Release Notes
Please add any useful notes about the feature/fix that might be helpful for others.

---
### Testing status
**Katana Tests**: First off we need to make sure the Katana SRP tests are green?

**Manual Tests**: What did you do?
Checked lightweight and HD shader generation.

**Automated Tests**: What did you setup?

Any test projects to go with this to help reviewers?

---
### Overall Product Risks
**Technical Risk**: None, Low, Medium, High?
Low
**Halo Effect**: None, Low, Medium, High?
Low
---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

**Comment by sebastienlagarde** [comment]
There is the stacklit master node now to add to the list :) (once your merge master)

**Comment by russoh** [comment]
Apparently the current tags for lightweight look weird, but actually do the right thing, so this isn't urgent to merge, but should probably happen soon anyway, since it's definitely hokey.

**Comment by sebastienlagarde** [comment]
Sorry for the dealy, haven't noticed the PR. Need to update Hair + Decal master :P

File: com.unity.render-pipelines.lightweight/Editor/ShaderPreprocessor.cs [hunk 1]
```diff
@@ -50,7 +50,7 @@ bool StripUnusedShader(ShaderFeatures features, Shader shader, ShaderCompilerDat
             if (shader.name.Contains("Debug"))
                 return true;
 
-            if (shader.name.Contains("HDRenderPipeline"))
+            if (shader.name.Contains("HDRP"))
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.lightweight/Editor/ShaderPreprocessor.cs]: Note: this is intented :)

**Comment by sebastienlagarde** [comment]
Note: I update the PR for all HDRP master node and change the name to match C# convention to k_ShaderTagName

Note that I make an attempt to rename LightweightPipeline to LWRP and HDRenderPipeline HDRP as we have been agree with the doc team to use that everywhere but it broke the preview (No idea why ??)

Merging as this PR have already been approved and need to move forward (cc @phi-lira )

**Comment by sebastienlagarde** [comment]
Note: I have fix the name to be the original one use in this PR for Lightweight here: https://github.com/Unity-Technologies/ScriptableRenderPipeline/pull/2685 sorry about that (Friday late night :P)

**Pull request merged**